mico-core/src/main/java/io/github/ust/mico/core/service/MicoKubernetesClient.java

```diff
@@ -226,6 +225,26 @@ public boolean isApplicationDeployed(MicoApplication micoApplication) throws Kub
         return result;
     }
 
+
+    /**
+     * Checks if a MICO service is already deployed.
+     *
+     * @param micoService the {@link MicoService}
+     * @return if true the servicegit s is deployed.
+     * @throws KubernetesResourceException if there is an error while retrieving the Kubernetes objects
+     */
+    public boolean isServiceDeployed(MicoService micoService) throws KubernetesResourceException {
```

Comment:
I would prefer the naming `isMicoServiceDeployed`. Otherwise it could be confusing whether the MICO service or the Kubernetes service is meant